packages/form-js-editor/src/render/components/icons/Draggable.svg

```diff
@@ -0,0 +1,14 @@
+<svg fill="currentcolor" version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" 
+	 width="16px" height="16px" viewBox="0 0 32 32" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:none;}
+</style>
+<title>draggable</title>
+<rect x="10" y="6" width="4" height="4"/>
+<rect x="18" y="6" width="4" height="4"/>
+<rect x="10" y="14" width="4" height="4"/>
+<rect x="18" y="14" width="4" height="4"/>
+<rect x="10" y="22" width="4" height="4"/>
+<rect x="18" y="22" width="4" height="4"/>
+<rect id="_Transparent_Rectangle_" class="st0" width="32" height="32"/>
+</svg>
```

Comment:
You can reduce a bit on the SVG size if you this one:

```svg
<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" width="16" height="16" fill="currentcolor" viewBox="0 0 32 32"><path d="M10 6h4v4h-4zm8 0h4v4h-4zm-8 8h4v4h-4zm8 0h4v4h-4zm-8 8h4v4h-4zm8 0h4v4h-4z"/><path d="M0 0h32v32H0z" style="fill:none"/></svg>
```

[Optimized via this website](https://jakearchibald.github.io/svgomg/)